guides/micronaut-scope-types/kotlin/src/main/kotlin/example/micronaut/Application.kt

```diff
@@ -0,0 +1,6 @@
+package example.micronaut
+
+import io.micronaut.runtime.Micronaut.run
+fun main(args: Array<String>) {
```

Comment:
this file is not necessary, it will be generated automatically. Please delete it.

Reply:
removed